### PR TITLE
WIP: Added tests

### PR DESCRIPTION
**Pull request by vanodevium**

### CODE REVIEW

File: src/cookie.js [hunk 1]
```diff
@@ -42,6 +42,12 @@ function setCookie(name, value, options = {path: '/'}) {
     - httpOnly (bool) - if true, cookie won't be available for using in JavaScript
                         IT CAN'T BE FALSE
   */
+  if (!name) {
+    return;
+  }
+
+  options = options || {};
```
Comment (UnSstrennen) [inline src/cookie.js:49]: ????

File: src/cookie.js [hunk 2]
```diff
@@ -70,3 +76,17 @@ function deleteCookie(name) {
     path: '/'
   })
 }
+
+if (typeof process !== 'undefined' &&
```
Comment (UnSstrennen) [inline src/cookie.js:80]: ????

**Review by UnSstrennen — COMMENTED** [review]
Please comment out changes in main cookie.js file: I can't understand what are you trying to do.

File: src/cookie.js (hunk 1 above)
Comment (vanodevium) [reply src/cookie.js:49]: If anyone pass null to options code is broken.
So we need to do this casting.

This is basic pattern for working with arguments without type-hinting or strict types

File: src/cookie.js (hunk 2 above)
Comment (vanodevium) [reply src/cookie.js:80]: This condition detect node environment and make library available for testing

**Pull request merged**